generalized_accounts/README.md

```diff
@@ -341,6 +341,11 @@ used instead.
 The `tx`, or inner transaction, MUST be a well formed transaction with nonce set
 to `0`. The `tx` MUST NOT be a `ga_attach_tx` transaction.
 
+The `authorization` function is executed in a special FATE environment where
+`Auth.tx_hash` and `Auth.tx` are available. These can (and should!) be used to
+securely authorize a transaction. Note: `Auth.tx_hash` computation differs
+between protocol versions, see [serialisation](../serializations.md)
```

Comment:
British or American spelling? The link says 'serializations' ... And 'authorize' is standard American spelling.

Reply:
Hmm, copy paste so someone else couldn't decide either 😅